src/Term/TermFallback.php

```diff
@@ -46,6 +46,18 @@ public function __construct( $requestedLanguageCode, $text, $actualLanguageCode,
 			throw new InvalidArgumentException( '$sourceLanguageCode must be a non-empty string or null' );
 		}
 
+		if ( $actualLanguageCode === $sourceLanguageCode ) {
```

Comment:
I'm not sure if this is a good idea. Why shouldn't a user of this class be allowed to use it like that if he wants?

Alternative proposal: Check if `source` is identical to `actual`, and if it is, set `source` to `null`.